src/HLL/SysConfig.nqp

```diff
@@ -0,0 +1,34 @@
+class HLL::SysConfig {
+    my %build-config := nqp::hash();
+    my $nqp-home;
+
+    method nqp-build-config() { %build-config };
+
+    method nqp-home() {
+        if !$nqp-home {
+            # Determine NQP directory
+            my $sep := nqp::backendconfig<osname> eq 'MSWin32' ?? '\\' !! '/';
```

Comment:
I think Configure.pl must put the separator into the hll-config when we generate it. Too often a line like this one appears in many places across the code.

Reply:
Moreover, SysConfig can provide basic filespec routines for the rest of the code. We can think about it later.

Reply:
I didn't retrieve it from the config but calculate it in place. That makes the logic not depend on the configuration. That doesn't force us to initialize the configuration before retrieving the separator and also we don't have to worry about the configuration to either be the NQP or Rakudo one.